packages/cli/src/commands/spaces/destroy.ts

```diff
@@ -43,7 +44,19 @@ export default class Destroy extends Command {
     if (space.state === 'allocated') {
       ({body: space.outbound_ips} = await this.heroku.get<Required<Heroku.SpaceNetworkAddressTranslation>>(`/spaces/${spaceName}/nat`))
       if (space.outbound_ips && space.outbound_ips.state === 'enabled') {
-        natWarning = `The Outbound IPs for this space will be reused!\nEnsure that external services no longer allow these Outbound IPs: ${displayNat(space.outbound_ips)}\n`
+        natWarning = heredoc`
+          ${ux.styledHeader('WARNING: Outbound IPs Will Be Reused')}
+          ${chalk.yellow('⚠️  The following outbound IPs (IPv4 and IPv6) will become available for reuse:')}
```

Comment:
Elsewhere in the CLI we're setting colors via the `@heroku-cli/color` package (which is essentially a passthrough to `chalk` with some preset colors for common heroku resources). For consistency, and in case we want to update things across the board from within the color package we should probably continue using it instead of going directly to chalk.

Reply:
okk. I wondered about that. I wasn't sure if this was going to need to be removed when we went to oclif 4 (I thought someone brought this up)

I'll use the `@heroku-cli/color` package

Reply:
done!